### ISSUE TITLE
Não funciona mais

### ISSUE THREAD

**Issue by ghost** [italoseara/whatsappy#21]
Olá Italo, tudo bem?
Quando tiver um tempo poderia atualizar a API. Ocorreram algumas mudanças recentes no sellenium nas chamadas de find_element.
Agradeço!

**Comment by italoseara** [comment]
certo, assim que eu tiver tempo, dou uma olhada, se puder me enviar a log dos erros, ajudaria muito

**Comment by ghost** [comment]
Não chega mais a aparecer que o whatsapp web is ready

**Comment by italoseara** [comment]
Houve uma atualização no selenium com relação aos drivers, já resolvi, porém, recentemente teve uma grande mudança na interface do whatsapp, mudando todos os seletores CSS, fazendo com que eu tenha que mudar todos, vai demorar um pouco, mas acredito que termino ainda hoje, e faço voltar a funcionar
